test/evergreen.yml

```diff
@@ -4082,9 +4032,12 @@ buildvariants:
     - name: compile
     - name: unit-test
     - name: format-smoke-test
-    - name: format-asan-smoke-ppc-test
+    - name: format-asan-smoke-test
     - name: format-wtperf-test
     - name: ".stress-test-ppc-1"
+    - name: ".stress-test-ppc-2"
+    - name: ".stress-test-ppc-3"
+    - name: ".stress-test-ppc-4"
```

Comment:
As PPC building resources are relatively limited in the system, running 2 instances of sanitizer tasks should be enough. Can we get `stress-test-ppc-3` and `stress-test-ppc-3` removed (here and above tagging place) accordingly? Thanks